spec/models/automation_request_spec.rb

```diff
@@ -88,6 +88,61 @@
     end
   end
 
+  context ".create_from_scheduled_task" do
+    let(:admin) { FactoryGirl.create(:user_miq_request_approver) }
+
+    it "with prescheduled task" do
+      ar = AutomationRequest.create_from_scheduled_task(admin, @uri_parts, @parameters)
+      expect(ar).to be_kind_of(AutomationRequest)
+      expect(ar).to eq(AutomationRequest.first)
```

Comment:
@bdunne - Validating that the new record has all the needed attributes expected of a scheduled job.  With the special addition of `AutomationRequest` we're bringing a number of additional requirements.  This test wraps those requirements into an easy to read format.